src/ngraph/runtime/gpu/cudnn_emitter.cpp

```diff
@@ -112,13 +112,57 @@ runtime::gpu::CUDNNEmitter::CUDNNEmitter(GPUPrimitiveEmitter* emitter)
 {
 }
 
+cudnnDataType_t runtime::gpu::CUDNNEmitter::getCudnnDataType(std::string dtype)
+{
+    // CUDNN_DATA_FLOAT
```

Comment:
The comments seem redundant. The map provides the same info. I am opening this for debate. Do you agree?

Reply:
agree.

Reply:
fixed.